index.js

```diff
@@ -56,16 +56,31 @@ class NpmAddDependencies {
 
   runNpmShow(dep) {
     return new Promise((resolve) => {
-      npmRun.exec(`npm show ${dep} dist-tags`, (err, stdout) => {
+      npmRun.exec(`npm show ${dep} dist dist-tags`, (err, stdout) => {
         if (!err) {
           const parsed = stdout.match(/latest: '(.*?)'/i);
 
           if (!parsed || undefined === parsed[1]) {
-            console.error(`Could not obtain the latest version for: ${dep}. Skip.`);
+            if (!dep.includes('@')) {
+              console.error(`Could not obtain the latest version for: ${dep}. Skip.`);
+            } else {
+              const [depName, depVersion] = dep.split('@');
```

Comment:
What about moving this above, before `return` statement, so we could use it everywhere inside promise callback?

Reply:
Thank you for idea, I changed source code, please see [this commit](https://github.com/arfeo/npm-add-dependencies/pull/5/commits/4c13451a27f688b9c6800e8f400f880da318ad28).